### PR TITLE
Initialize setup

### PR DESCRIPTION
**Pull request by karanJ2212**
In this milestone I have

-initialized react app
-installed routers
-created components
-installed linters
-followed gitflow

### CODE REVIEW

File: src/components/Addbook.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+      <h2>Add a Book</h2>
+      <input
+        type="text"
+        placeholder="
+      title of book"
+      />
+      <input type="text" placeholder="Author of book" />
+    </div>
+  );
+}
```
Comment (mckent05) [inline src/components/Addbook.js]: - According to the project requirements, this component should have a button to submit (the button wouldn't add the book yet, this is just a placeholder for now). Please fix this 🙏

File: src/components/Books.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  return (
+    <div className="bookContainer">
+      <ul className="booklist">
+        {books.map((book) => (
+          <li key={book.title + book.autor}>
+            <Book title={book.title} author={book.author} />
+
+            <button className="remove" type="button">
+              remove
+            </button>
```
Comment (mckent05) [inline src/components/Books.js]: -There is already a `remove` button in the `Book` component, please remove this button if it is not needed so you have only what is required on your app. Please fix this 😄

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @karanJ2212, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Addbook.js (hunk 1 above)
Comment (karanJ2212) [reply src/components/Addbook.js]: ok

**Review by TadesseDev — APPROVED** [review]
### Hi @karanJ2212 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
68747470733a2f2f6669636f666f72756d732e6d796669636f2e636f6d2f74352f696d6167652f736572766572706167652f696d6167652d69642f343932363069463839303835363833444139334135422f696d6167652d73697


## Highlights
 - [ ] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**